### DISCUSSION TITLE
Disable "no thumbnail" screen?

### DISCUSSION THREAD

**Discussion by gthb96** [mriscoc/Ender3V2S1#1160] [Q&A]
### Did you test with a precompiled firmware?

Yes, and the problem still exists.

### Bug Description

Not a bug...

I am (almost) sure it was somehow possible to disable the confirmation screen after file selection maybe until mid of this year.
As I didn't configure thumbnail generation in PrusaSlicer (atleast I am almost sure) I had the confirmation screen disabled in the past. (99% sure)

I cannot seem to find a setting for that anymore, maybe it was even a slicer setting that skipped this dialog but now after an hour of trying useless things I really want to know if this was a dream or how I did that before.

Thank you already and thank you for this project!

### Version of Professional Firmware

Ender3V2-422-MM-MPC-20230904

### Printer model

Ender3V2

### Electronics

Stock 4.2.2

### Bed Leveling

None

### Your Slicer

Prusa Slicer

### Host Software

SD Card (headless)

**Comment by mriscoc** [comment]
For remove the confirmation screen the firmware must be compiled with HAS_GCODE_PREVIEW disabled or with PREVIEW_MENU_ITEM enabled to be able to enable/disable the preview via the menu.

**Comment by gthb96** [comment]
Thank you for pointing that out, will build later!
But, was this enabled in previous prebuilt-firmwares?

**Reply by mriscoc** [reply]
Some versions ago, PREVIEW_MENU_ITEM  was enabled by default, but due to lack of program memory space in F4 SoC, it was removed.
